lib/SiteConfig.py

```diff
@@ -342,6 +342,7 @@ def interpolate_jinja(self):
 
             # don't render defaults
```

Comment:
comment seems obsolete

Reply:
comment is still correct.  we don't _render_ them.